src/System Application/App/Page Style/PageStyle.al

```diff
@@ -0,0 +1,49 @@
+enum 50000 "Page Style"
+{
+    Extensible = false;
+
+    value(0; None)
+    {
+        Caption = 'None', Comment = 'None';
+    }
+    value(1; Standard)
+    {
+        Caption = 'Standard', Comment = 'Standard';
+    }
+    value(2; StandardAccend)
+    {
+        Caption = 'StandardAccent', Comment = 'Blue';
```

Comment:
The `Comment` property is for translators of the `Caption` property; it is exported when you generate the g.xlf.
Since there is no Description property available for enum values, you could put it into comments.

Reply:
added the comments just like in the real options, so you have the explanation at hover. Thank you for pointing out